packit_service/models.py

```diff
@@ -889,19 +889,26 @@ def get_by_build_id(
             return query.first()
 
     @classmethod
-    def get_all_by_owner_and_project_and_target(
+    def get_all_by_owner_project_target_commit(
         cls,
         owner: str,
         project_name: str,
         target: str,
+        commit_sha: str,
```

Comment:
Shouldn't commit sha and project name (just to make sure, but I think at least GitHub is switched to new SHA after the collision was discovered) be enough?

Reply:
I'm not sure how do you mean it, which parameters do you suggest removing?

Reply:
Keeping only `target` and `commit_sha`, but it's just a thought; It is safer the way it is :)